lighthouse-core/driver/index.js

```diff
@@ -100,15 +100,19 @@ class Driver {
     const loadData = {};
     let pass = Promise.resolve();
 
+    if (config.traceName) {
+      loadData[config.traceName] = {};
```

Comment:
if you instead add `traceContents` and `networkRecords` to `tracingData[config.traceName]` at https://github.com/GoogleChrome/lighthouse/blob/94cc318ee4b02ae768e4ef4e6ec85203a82442a0/lighthouse-core/driver/index.js#L167 and leave `loadData` unaltered here, gatherers won't need to know the `traceName` of the pass they're in (which makes sense, as they should only ever have access to the trace data for the pass they're in)